source/includes/_wallet.md

```diff
@@ -1,13 +1,39 @@
 # Wallet
 ## The WalletDB and Object
+```javascript
+let id, url;
+```
+
+```shell--vars
+id="primary"
+url="http://localhost:18332"
+```
+
+```shell--curl
+curl $url/wallet/primary/
+```
+
+```shell--cli
+bcoin cli wallet get
```

Comment:
Could we add `--id=primary` here, and update the relevant doc section that `--id` defaults to primary if not passed.

I just wanted to mention this here since the concept of multiple wallets and selecting a wallet from the `cli` is a bit unclear to the beginner.

Reply:
Added to the example and added explanation in the getWalletInfo section